diff --git a/dist/index.d.ts b/dist/index.d.ts
index 94fb88c5d8dfc9bec0876230744e0082287f9c9f..4eb2cea6b2ee4213e92587cc16ddb7555b52f821 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -7,11 +7,11 @@ import type * as Layer from "effect/Layer";
 import type * as Schema from "effect/Schema";
 import type * as Scope from "effect/Scope";
 import type * as FC from "effect/testing/FastCheck";
-import * as V from "vitest";
+import * as V from "@voidzero-dev/vite-plus-test";
 /**
  * @since 1.0.0
  */
-export * from "vitest";
+export * from "@voidzero-dev/vite-plus-test";
 /**
  * @since 1.0.0
  */
diff --git a/dist/index.js b/dist/index.js
index 32a4f81e4eb6883231b47ec2ac30ffad86595b5a..a24142fb5c330f2aed392225d147dc0ec3f36f10 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,9 +1,9 @@
-import * as V from "vitest";
+import * as V from "@voidzero-dev/vite-plus-test";
 import * as internal from "./internal/internal.js";
 /**
  * @since 1.0.0
  */
-export * from "vitest";
+export * from "@voidzero-dev/vite-plus-test";
 /**
  * @since 1.0.0
  */
diff --git a/dist/internal/internal.js b/dist/internal/internal.js
index 7fd3a9d25e17e474a88ce137e5dc164a3d915c3c..04732953a1a6749d43622d58ed4b97f3a1f2a00e 100644
--- a/dist/internal/internal.js
+++ b/dist/internal/internal.js
@@ -14,7 +14,7 @@ import * as Scope from "effect/Scope";
 import * as fc from "effect/testing/FastCheck";
 import * as TestClock from "effect/testing/TestClock";
 import * as TestConsole from "effect/testing/TestConsole";
-import * as V from "vitest";
+import * as V from "@voidzero-dev/vite-plus-test";
 const runPromise = /*#__PURE__*/Effect.fnUntraced(function* (effect, _ctx) {
   const exit = yield* Effect.exit(effect);
   if (Exit.isFailure(exit)) {
diff --git a/dist/utils.js b/dist/utils.js
index 72dfe8217daaab11d33b3c255d81fb33d56c9bcf..4472f4b5d2a2eb1c3169312f2384a68eea6443df 100644
--- a/dist/utils.js
+++ b/dist/utils.js
@@ -4,7 +4,7 @@ import * as Option from "effect/Option";
 import * as Predicate from "effect/Predicate";
 import * as Result from "effect/Result";
 import * as assert from "node:assert";
-import { assert as vassert } from "vitest";
+import { assert as vassert } from "@voidzero-dev/vite-plus-test";
 // ----------------------------
 // Primitives
 // ----------------------------
